router/core/errors.go

```diff
@@ -122,10 +122,14 @@ func propagateSubgraphErrors(ctx *resolve.Context, logger *zap.Logger) {
 // It accepts a graphqlerrors.RequestErrors object and writes it to the response based on the GraphQL spec.
 func writeRequestErrors(r *http.Request, w http.ResponseWriter, statusCode int, requestErrors graphqlerrors.RequestErrors, requestLogger *zap.Logger) {
 	if requestErrors != nil {
-		if statusCode != 0 {
-			w.WriteHeader(statusCode)
-		}
+
 		if r.URL.Query().Has("wg_sse") {
```

Comment:
did you not want to fix this? =)

Reply:
Yes, but in a separate PR. This is already too big.